web/modules/custom/dpl_opening_hours/dpl_opening_hours.module

```diff
@@ -47,6 +52,79 @@ void {
   }
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter() for the taxonomy term deletion form related to opening hours.
+ *
+ * This hook modifies the deletion confirmation form for taxonomy terms within
+ * the "opening_hours_categories" vocabulary, adding custom messages.
+ *
+ * @param array $form
+ *   The form structure array.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The current state of the form.
+ * @param string $form_id
+ *   The ID of the form being altered.
+ */
+function dpl_opening_hours_form_taxonomy_term_opening_hours_categories_delete_form_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
+  $form_object = $form_state->getFormObject();
+  if (!$form_object instanceof EntityFormInterface) {
+    return;
+  }
+
+  $term = $form_object->getEntity();
+  if (!$term instanceof Term) {
+    return;
+  }
+
+  $term_id = $term->id();
+  if (!is_int($term_id) && !is_string($term_id)) {
+    return;
+  }
+
+  $term_label = $term->label();
+
+  $form['#title'] = t('Are you sure you want to delete all "@term_label" opening hours?', [
+    '@term_label' => $term_label,
+  ]);
+
+  $opening_hours_repository = DrupalTyped::service(OpeningHoursRepository::class, 'dpl_opening_hours.repository');
+  $opening_hours_in_category = $opening_hours_repository->loadMultiple(categoryId: (int) $term_id);
+  $opening_hours_count = count($opening_hours_in_category);
+
+  $form['description']['#markup'] = t('@term_label is associated with @count opening hours. Deleting @term_label will also delete these opening hours. Are you sure you want to continue?', [
+    '@term_label' => $term_label,
+    '@count' => $opening_hours_count,
```

Comment:
```suggestion
  $form['description']['#markup'] = t('@term_label is associated with @count opening hours. Deleting @term_label will also delete these opening hours. Are you sure you want to continue?', [
    '@term_label' => $term_label,
    '@count' => count($opening_hours_in_category),
```

Consider not extracting everything to a separate variable.

Especially if extracting it clearly shows what the value contains. If a value is used multiple times (like `$term_label`) or your lines become very long then I understand it but that is not the case here.